ondemand.go

```diff
@@ -3,40 +3,33 @@ package traefik_ondemand_plugin
 import (
 	"context"
 	"fmt"
-	"io/ioutil"
-	"log"
 	"net/http"
-	"strings"
 	"time"
 
-	"github.com/acouvreur/traefik-ondemand-plugin/pkg/pages"
+	"github.com/acouvreur/traefik-ondemand-plugin/pkg/strategy"
 )
 
-// Net client is a custom client to timeout after 2 seconds if the service is not ready
-var netClient = &http.Client{
-	Timeout: time.Second * 2,
-}
-
 // Config the plugin configuration
 type Config struct {
 	Name       string `yaml:"name"`
 	ServiceUrl string `yaml:"serviceurl"`
 	Timeout    string `yaml:"timeout"`
+	WaitUi     bool   `yaml:"wait-ui"`
+	BlockDelay string `yaml:"block-delay"`
```

Comment:
I've tested this branch, looks like the hyphen is not allowed in some way. I can confirm the following works:

```go
	WaitUi     bool   `yaml:"waitui"`
	BlockDelay string `yaml:"blockdelay"
```
with config
```yaml
          waitui: false
          blockingdelay: 20s
```

Otherwise WaitUi will remain true based on the default config from `CreateConfig`

Reply:
It should work with hyphens, so I should find out why it doesn't